_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -87,3 +87,28 @@ We highly recommend reviewing the detailed list of breaking changes in our docum
 We advise all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
 
 Get ready to unleash your creativity and build impactful decentralized applications with the latest version of the Solidity Compiler. Happy coding!
+
+## New linux binaries released for Solidity 0.8.18
```

Comment:
I think we should make it clear this is an update to the original post (i.e. something you don't have to worry about if you did not update to 0.8.18 yet).

The text says that but I think that including something like `UPDATE` in the header would be clearer.

```suggestion
## UPDATE 2022-02-02: Rebuilt Linux binary for Solidity 0.8.18
```